Automattic/wp-calypso
# Checkout: show transfer progress during ecommerce plan checkout

#### Changes proposed in this Pull Request

* Replaces the current loading screen with [the signup loading screen](https://github.com/Automattic/wp-calypso/issues/53911#issuecomment-880639205)
* Shows 3 steps: Upgrading your site, Installing plugin, Activating plugin
* The animation is timed to last one minute

Demo:
Only 2 steps before transfer completes - https://www.loom.com/share/3af8b58f16104872aaaa021ffc591448
3 old steps - https://www.loom.com/share/6cbd0115f5454c37afb1398b7a3b4793

Test live:
https://calypso.live?image=registry.a8c.com/calypso/app:build-11895

Screenshots:
<img width="920" alt="Screen Shot 2021-08-04 at 9 58 38 AM" src="https://user-images.githubusercontent.com/1689238/128208024-a49749ba-580b-4f33-b935-9bde03605ebe.png">
<img width="752" alt="Screen Shot 2021-08-04 at 11 23 45 AM" src="https://user-images.githubusercontent.com/1689238/128208325-e43c3c83-075c-418e-97ad-51352d0009a9.png">
<img width="732" alt="Screen Shot 2021-08-04 at 11 23 05 AM" src="https://user-images.githubusercontent.com/1689238/128208364-96fc50e4-71de-4af5-99f8-f5539adf9193.png">


#### Testing instructions



* Purchase a new site with eCommerce plan at /new
* Observe the loading screen after the checkout



Related to https://github.com/Automattic/wp-calypso/issues/53911
